core/domain/rights_manager_test.py

```diff
@@ -783,23 +856,78 @@ def test_deassign_owner_is_successful(self):
         exp_rights = rights_manager.get_exploration_rights(self.EXP_ID)
         self.assertFalse(exp_rights.is_owner(self.user_id_b))
 
-    def test_deassign_non_existent_fails(self):
+    def test_deassign_non_existent_fails(self) -> None:
         exp = exp_domain.Exploration.create_default_exploration(self.EXP_ID)
-        exp_services.save_new_exploration(self.user_id_a, exp)
+        exp_services.save_new_exploration(self.user_id_a, exp)  # type: ignore[no-untyped-call]
 
-        with self.assertRaisesRegex(
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
             Exception, 'This user does not have any role in'):
             rights_manager.deassign_role_for_exploration(
                 self.user_a, self.EXP_ID, self.user_id_b)
 
+    def test_deassign_editor_is_successful_with_all_valid_commit_messages(
+        self
+    ) -> None:
+        editor_username = 'B'
+        self.assertEqual(
+            user_services.get_username(self.user_id_b),
+            editor_username
+        )
+
+        # Creating new exploration.
+        exp = exp_domain.Exploration.create_default_exploration(self.EXP_ID)
+        exp_services.save_new_exploration(self.user_id_a, exp)  # type: ignore[no-untyped-call]
+
+        snapshots_data = (
+            exp_models.ExplorationRightsModel.get_snapshots_metadata(
+                self.EXP_ID, [1]
+            )
+        )
+        self.assertEqual(
+            snapshots_data[0]['commit_message'],
+            'Created new exploration'
+        )
+
+        # Assigning editor role to editor_username ('B').
```

Comment:
Given the error that happened previously, I suggest you use a multi-character username here.

Reply:
Done ~ new username is `TestUser`